packages/definitions/dist/fhir/r4/search-parameters-medplum.json

```diff
@@ -1326,7 +1326,7 @@
         "id": "AsyncJob-status",
         "url": "https://medplum.com/fhir/SearchParameter/AsyncJob-status",
         "version": "4.0.1",
-        "name": "type",
+        "name": "status",
```

Comment:
Was this intentional to commit? We should probably also run generator to put that into the docs